xml/art_solution_monitoring.xml

```diff
@@ -0,0 +1,251 @@
+-->
+
+<article version="5.1" xml:lang="en" xml:id="art-sol-monitoring"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+   <title>&sap; Monitoring</title>
+   <info>
+     <abstract>
+       <para>
+         This article shows monitoring solutions for &sap; administrators
+         to efficiently monitor their &sap; systems.
+       </para>
+       <para>
+         The solution described herein is not officially supported by &suse;.
+         Nevertheless, it is a solution based on best-efforts and published
+         <quote>as is</quote>.
+       </para>
+     </abstract>
+   </info>
+
+  <sect1 xml:id="sec-sol-monit-concept-overview">
+    <title>Conceptual Overview</title>
+    <para>
+      Starting from the idea to improve user experience, &suse; engineering worked
+      on how to monitor and troubleshoot clusters.
+    </para>
+    <para>
+      The exporters showed here can be combined and integrated with Prometheus
+      and Grafana to produce complex dashboards.
+    </para>
+    <para>
+      &suse; supports Prometheus and Grafana through &suma; 4.0. Some Grafana
+      dashboards for &hana;, &s4h;, &netweaver;, and the cluster monitoring
+      are provided by &suse; via Grafana community dashboards.
+    </para>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-monit-terms">
+    <title>Terminology</title>
+    <para></para>
+    <variablelist>
+      <varlistentry>
+        <term>Grafana</term>
+        <listitem>
+          <para>
+            An interactive, visualization and analytics web application.
+            With dashboards, it provides methods to visualize, explore,
+            alert on, and query your metrics.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Prometheus</term>
+        <listitem>
+          <para>
+            A systems monitoring and alerting toolkit. It collects and evaluates
+            metrics, displays the result, and trigger possible alerts
+            when a observed condition is true.
+            Metrics can be collected from different targets at given intervals.
+          </para>
+        </listitem>
+      </varlistentry>
+    </variablelist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-monit-hana-db-exp">
+    <title>&hana; Database Exporter</title>
+    <para>
+      Export &hana; database metrics.
+      The exporter is able to export the
+      metrics from more than one database or tenant. This option
+      (<option>multi_tenant</option>) is enabled by default.
+    </para>
+    <formalpara>
+      <title>Package:</title>
+      <para><package>prometheus-hanadb_exporter</package></para>
+    </formalpara>
+    <formalpara>
+      <title>Link:</title>
+      <para>
+        See <link xlink:href="https://github.com/SUSE/hanadb_exporter"/>.
+      </para>
+    </formalpara>
+    <itemizedlist>
+      <title>Export Metrics in Prometheus Format</title>
```

Comment:
Should this read 'Export' (noun) or 'Exports' (verb)?

Reply:
The latter (verb). Fixed.